crypto/bn/bn_mont.c

```diff
@@ -258,6 +258,8 @@ int BN_MONT_CTX_set(BN_MONT_CTX *mont, const BIGNUM *mod, BN_CTX *ctx)
     R = &(mont->RR);            /* grab RR as a temp */
     if (!BN_copy(&(mont->N), mod))
         goto err;               /* Set N */
+    if (BN_get_flags(mod, BN_FLG_CONSTTIME) != 0)
+        BN_set_flags(&(mont->N), BN_FLG_CONSTTIME);
```

Comment:
I'm still thinking that it would be better to do this in BN_copy() (just for this one flag). I know we still need to do it explicitly for `tmod` below, but I think that's ok. Did you look at doing this?